local-storage/create-local-persistent-volume.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash 
+if [ -z ${1} ];
+then 
+  echo "Usage: ./$0 create"
+  echo "to create pv ./$0 create"
+  echo "to delete pv ./$0 delete"
+  exit $?
+fi
+
+action=${1}
+
+external_volumes=$(ls -alth /dev/sd*  | grep -v sda | awk '{print $10}')
```

Comment:
to be safe, check if the volume has any label on it first before using it. otherwise it could destroy the existing volume and data

Reply:
e.g. using [blkid](https://linux.die.net/man/8/blkid), 
```console
# blkid /dev/vdb
# blkid /dev/vda
/dev/vda: PTUUID="71d0db04" PTTYPE="dos"
# blkid /dev/vda1
/dev/vda1: UUID="6575ffc7-e3a4-430c-9dbf-5cbd9b7ae1d6" BLOCK_SIZE="4096" TYPE="ext4" PARTUUID="71d0db04-01"
```